adapters/mobfoxpb/mobfoxpb.go

```diff
@@ -28,21 +39,40 @@ func Builder(bidderName openrtb_ext.BidderName, config config.Adapter) (adapters
 // MakeRequests create bid request for mobfoxpb demand
 func (a *adapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
 	var errs []error
-	var err error
 	var tagID string
-
+	var err_tag error
+	var key string
+	var err_key error
+	var route string
+	var method string
 	var adapterRequests []*adapters.RequestData
 
+	requestURI := a.URI
 	reqCopy := *request
 	imp := request.Imp[0]
-	tagID, err = jsonparser.GetString(imp.Ext, "bidder", "TagID")
-	if err != nil {
-		errs = append(errs, err)
+	tagID, err_tag = jsonparser.GetString(imp.Ext, "bidder", "TagID")
+	key, err_key = jsonparser.GetString(imp.Ext, "bidder", "key")
+	if err_tag != nil && err_key != nil {
```

Comment:
This will only trigger if both errors out. I think you want to use or (||) here.

Reply:
If this wasn't cause by a test case, there may be a testing gap.

Reply:
No that's valid, we except etleast one, and fail only if there is now neither tag nor key